docs/trellis/master/ssh-keys.md

```diff
@@ -148,5 +148,5 @@ The Trellis `ansible.cfg` file enables this SSH agent forwarding with `ssh_args
 Remember to import your SSH key password into Keychain by running:
 
 ```bash
-$ ssh-add -K
+$ ssh-add --apple-use-keychain
```

Comment:
We should probably caveat this, since the deprecation only started with Monterey.

```suggestion
# macOS 12.0 and later:
$ ssh-add --apple-use-keychain
# Prior versions of macOS:
$ ssh-add -K
```

Reply:
I’ll take a look through the docs to see if there’s a different way we do this sort of thing for anything.

Reply:
I reworded it here: [583321d](https://github.com/roots/docs/pull/398/commits/583321d354536b33314ded273c42a1ef61b61f64)